src/meta/processors/schemaMan/AlterEdgeProcessor.cpp

```diff
@@ -88,11 +91,20 @@ void AlterEdgeProcessor::process(const cpp2::AlterEdgeReq& req) {
     }
     schema.set_columns(std::move(columns));
 
-    std::vector<kvstore::KV> data;
     LOG(INFO) << "Alter edge " << req.get_edge_name() << ", edgeType " << edgeType;
     data.emplace_back(MetaServiceUtils::schemaEdgeKey(spaceId, edgeType, version),
                       MetaServiceUtils::schemaEdgeVal(req.get_edge_name(), schema));
     resp_.set_id(to(edgeType, EntryType::EDGE));
+    // Now we get default value from meta instead of cache
```

Comment:
Is the annotation wrong?

Reply:
Don't, graph get it from meta directly now, I had a PR #2011 to fix it.